stratum/hal/lib/barefoot/bf_chassis_manager.cc

```diff
@@ -28,23 +27,23 @@ namespace stratum {
 namespace hal {
 namespace barefoot {
 
-using PortStatusChangeEvent = BFPalInterface::PortStatusChangeEvent;
+using LinkscanEvent = BfSdeInterface::LinkscanEvent;
```

Comment:
I think "linkscan" is a term that Broadcom uses in their SDK, and isn't necessarily standardized.

I'm okay with the name if you want to have commonality between targets, but personally, I find PortStatusChangeEvent to be more descriptive.

Reply:
The SDE seems to use port status notification or change. Commonality in the wrappers would be partial at best anyway, so not something we should pursue.

I'd still drop the `Change` part of the event name and fix the `VerbNoun` naming of some of the functions.

Reply:
Looks good after the change.

We should keep the terminologies separate, e.g. `unit` vs. `device`.